src/panels/UserPanelsPanel/UserPanelsPanel.tsx

```diff
@@ -0,0 +1,134 @@
+import { useEffect, useState } from 'react';
+import {
+  Button,
+  Container,
+  Divider,
+  Group,
+  Select,
+  Text,
+  TextInput,
+  Title
+} from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { UserPageIcon } from '@/icons/icons';
+import { useAppDispatch, useAppSelector } from '@/redux/hooks';
+import { intializeUserPages } from '@/redux/userpages/userpagesSlice';
+import { useWindowManagerProvider } from '@/windowmanagement/WindowLayout/hooks';
+
+import { UserPage } from './UserPanel';
+
+export function UserPanelsPanel() {
+  const { addWindow } = useWindowManagerProvider();
+
+  const [selectedPanel, setSelectedPanel] = useState<string | null>(null);
+  const [panelURL, setPanelURL] = useState<string>('');
+  const [urlPanelTitle, setUrlPanelTitle] = useState<string>('');
+
+  const luaApi = useOpenSpaceApi();
+  const isDataInitialized = useAppSelector((state) => state.userPages.isInitialized);
+
+  const localPanels = useAppSelector((state) => state.userPages.panels || []);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    // Collect all folder paths in the USER folder
+    const getUserPanels = async () => {
+      if (!luaApi) return;
```

Comment:
Lots of one line returns in this PR. Up until now we have avoided those and always used braces, i.e.
```suggestion
      if (!luaApi) {
        return;
      }
```
We should decide which version we prefer and stick to one. 

One thing to consider is that this would not be allowed according to our C++ coding guidelines, so if we want to be a similar to those as possible, we should used the braced version

Reply:
I'd prefer that we stick with braced version